src/utils/source-map-worker.js

```diff
@@ -18,6 +18,9 @@ const {
   isOriginalId
 } = require("./source-map-util");
 
+const { setConfig } = require("devtools-config");
+setConfig(DebuggerConfig);
```

Comment:
Are you sure this works in the panel? I remember trying to load the config before but ran into problems so I avoided it.